Bottom: 20b98a674cc90fb74869e152c8a4056f4566b144
Top:    0fe28d05aaadf63b88bc35c3ed409a656c5beb46
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 17:55:53 +0100

Refresh of w-c-split-gathering-routines-f

---

diff --git a/src/draw.c b/src/draw.c
index 08ea7e4..a4acfdf 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -873,12 +873,12 @@ draw_arc (LayerType *layer, ArcType *arc)
   if (TEST_FLAG (SELECTEDFLAG | FOUNDFLAG, arc))
     {
       if (TEST_FLAG (SELECTEDFLAG, arc))
-        gui->set_color (Output.fgGC, Layer->SelectedColor);
+        gui->set_color (Output.fgGC, layer->SelectedColor);
       else
         gui->set_color (Output.fgGC, PCB->ConnectedColor);
     }
   else
-    gui->set_color (Output.fgGC, Layer->Color);
+    gui->set_color (Output.fgGC, layer->Color);
 
   _draw_arc (arc);
 }
